### PR TITLE
[cancel] Update changelog_en.txt

### PR DESCRIPTION
**Pull request by ZorroGuevara**
3.7e and others small fixes
Don't forget to put the date when you release it ==> 3.7e 202X-XX-XX

### CODE REVIEW

File: translation/changelog_en.txt [hunk 1]
```diff
@@ -191,3328 +183,3 @@
 	Swipe left - to delete file
 - Direct download without site support for some file, .mp3/4 and .jpeg can now be downloaded whatever the site (#3935)
 - Other small things
-
```
Comment (KurtBestor) [inline translation/changelog_en.txt]: Don't remove these lines

File: translation/changelog_en.txt [hunk 2]
```diff
@@ -100,7 +100,7 @@
 
 
 --------------------------------------------------------------------------------------------------------------------------------------------
-3.7b 2021-10-29
```
Comment (KurtBestor) [inline translation/changelog_en.txt]: v3.7b has released at Oct 29, 2021. Why are you changing it?

File: translation/changelog_en.txt [hunk 3]
```diff
@@ -109,15 +109,7 @@
 
 
 --------------------------------------------------------------------------------------------------------------------------------------------
-3.7a 2021-10-28
```
Comment (KurtBestor) [inline translation/changelog_en.txt]: Why are you removing these lines?

File: translation/changelog_en.txt [hunk 4]
```diff
@@ -100,7 +100,7 @@
 
 
 --------------------------------------------------------------------------------------------------------------------------------------------
-3.7b 2021-10-29
+3.7b 2021-10-21
```
Comment (KurtBestor) [inline translation/changelog_en.txt]: v3.7b has been released at Oct 29, 2021. Why are you changing it?

**Comment by ZorroGuevara** [comment]
Sorry, bad copy-paste.

**Pull request closed without merging**